client/src/main/java/io/pravega/client/stream/impl/ControllerImpl.java

```diff
@@ -232,20 +233,31 @@ public ControllerImpl(ManagedChannelBuilder<?> channelBuilder, final ControllerI
     public AsyncIterator<Stream> listStreams(String scopeName) {
         Exceptions.checkNotClosed(closed.get(), this);
         long traceId = LoggerHelpers.traceEnter(log, "listStreams", scopeName);
```

Comment:
Do we still need this `traceId`?

Reply:
That is tracing the method entry and will match will corresponding method exit